src/lib/components/BookSelector.svelte

```diff
@@ -113,24 +132,36 @@ The navbar component.
         nextRef.reset();
     }
 
-    /**list of books in current docSet*/
+    /**list of books, quizzes, and quiz groups in current docSet*/
     $: books = $refs.catalog.documents;
+    $: quizzes = $refs.catalog.quizzes;
```

Comment:
We are not using this, correct?

Reply:
This is correct. I thought I deleted that.